inc/search.class.php

```diff
@@ -7703,20 +7888,22 @@ static function sylk_clean($value) {
    /**
     * Create SQL search condition
     *
-    * @param string  $field  Nname (should be ` protected)
+    * @param string  $field  Name (should be ` protected)
     * @param string  $val    Value to search
     * @param boolean $not    Is a negative search ? (false by default)
     * @param string  $link   With previous criteria (default 'AND')
     *
     * @return search SQL string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function makeTextCriteria ($field, $val, $not = false, $link = 'AND') {
+   public function makeTextCriteria ($field, $val, $not = false, $link = 'AND') {
```

Comment:
1. Should be mentionned in changelog.

2. This method is only privately used, may be changed to private.